### PR TITLE
Eliot 0.6 changes the way task levels are structured

### LINKED ISSUE #41: Make tests pass with latest Eliot
There's a failing test against 0.6 version of Eliot.

### PR DESCRIPTION
**Pull request by itamarst**
Fixes #41.

### CODE REVIEW

File: machinist/test/test_fsm.py [hunk 1]
```diff
@@ -868,7 +868,6 @@ def test_terminalLogging(self, logger):
                     u"action_type": u"fsm:initialize",
                     u"action_status": u"succeeded",
                     u"task_uuid": logger.messages[0][u"task_uuid"],
-                    u"task_level": u"/",
```
Comment (exarkun) [inline machinist/test/test_fsm.py]: Maybe something about `task_level` should still be logged here?  Or ... `action_type`, `action_status`, and `task_uuid` are sufficient to prove it's the right event and if `task_level` isn't what we expect it must be a bug in eliot?

Comment (itamarst) [inline machinist/test/test_fsm.py]: Mmmm. Or maybe we could switch to using one of the higher level Eliot testing APIs that will parse the whole action for you.

Comment (exarkun) [inline machinist/test/test_fsm.py]: Like `LoggedAction.of_type`?  Did something with that, see if you like it.

Comment (exarkun) [inline machinist/test/test_fsm.py]: Feel free to merge if so.

**Pull request merged**